plugins/pluginmanagers/lz-n.nix

```
@@ -146,6 +149,42 @@ lib.nixvim.plugins.mkNeovimPlugin {
       };
     in
     {
+      keymaps = mkOption {
+        type = types.listOf (mkMapOptionSubmodule {
+          extraOptions.plugin = mkOption {
+            type = with types; either str lzPluginType;
+            example = "telescope.nvim";
+            description = ''
+              The name of the plugin to lazy-load when the keymap is triggered.
+              This must match the name used in the `plugins` list or be a valid plugin name that can be loaded.
```

We could emphasize this as a note, warning, etc?

```suggestion

              > [!NOTE]
              > The name must match the name used in the `plugins` list or be a valid plugin name that can be loaded.
```